controllers/marrainageController.js

```diff
@@ -31,13 +31,19 @@ function getDataFromToken(token) {
   if (!data) {
     throw new Error('Corrupted data in token');
   }
-  return data;
+  const newcomerId = data.newcomerId;
```

Comment:
tu as moyens d'être retro compatible avec le token déjà envoyé ?

```suggestion
  const newcomerId = data.newcomerId || data.newcomer.id;
```

Reply:
Bien vu ! Du coup j'ai modifié (commit 62664c3c4c096d177560e0ce900eaa39c8a90dac) pour ne pas appeler les userinfos si l'ancien token est utilisé.